src/api.rs

```diff
@@ -357,6 +365,12 @@ impl Default for PixelRange {
     }
 }
 
+/// Contain different types of Matrix Coefficients which are used for deriving the luma and chroma (YCbCr) from the R, G, B primaries.
+/// - This is used to describe the color parameters for the video to be encoded, the default value is `Unspecified`.
+/// - Matrix Coefficients can also covert across a multitude of systems.
```

Comment:
same problem here, also what is this line for here? it should be coalesced in the first one

Reply:
```
[14:54:30] <koda>	the only major problem is that information is too sparse
[14:54:37]  <koda>	first there should be the most important thing
[14:54:41]    <koda>	then any ancillary information
[14:54:46]  <koda>	then tiny details if at all
[14:55:05]  <koda>	Matrix Coefficients is a ripe example
[14:55:11]  <koda>	 /// Contain different types of Matrix Coefficients which are used for deriving the luma and chroma (YCbCr) from the R, G, B primaries. 
[14:55:46]  <koda>	besides that it is not correct (or at least incomplete)
[14:55:59]  <koda>	the important part is actually the second line
[14:56:05]  <koda>	This is used to describe the color parameters for the video to be encoded, the default value is `Unspecified`.
[14:56:29]  <koda>	and the third sentence could be merged in the second one
[14:57:51]  <koda>	 // This is used to describe matrix coeffiencnts in the color parameters, the default value is `Unspecified`.   These are used to convert across a multitude of color systems, such as YCbCr to RGB and vice versa
```